src/test/regress/.gitignore

```diff
@@ -6,6 +6,7 @@
 /tmp_upgrade/
 /tmp_citus_upgrade/
 /tmp_citus_tarballs/
+/build/
 /results/
```

Comment:
Should this also contain `.deps`?

Reply:
I guess we already have it here https://github.com/citusdata/citus/blob/master/.gitignore#L15